openpype/plugins/publish/extract_burnin.py

```diff
@@ -281,14 +307,16 @@ def main_process(self, instance):
             # NOTE we maybe can keep source representation if necessary
             instance.data["representations"].remove(repre)
 
-            # Delete input files
```

Comment:
this has to be returned back, there is certainly duplicity in representations if this is not letting rest of the process to go on

Reply:
altaught this had been reveresed and it is reflected in the code it is not displayed here in the changes so be aware that this is not showing correct state..

Reply:
this is the commit reversing it.. [`dd27953` (#2248)](https://github.com/pypeclub/OpenPype/pull/2248/commits/dd27953068efe35579ae77fb3401e539ffaef041)